naughts_crosses.py

```diff
@@ -30,6 +30,30 @@ def prepareBoard():
 
     return board
 
+def drawBoard():
+    global board
+    global COLUMNS
+    global ROWS
+
+    print('| 1 | 2 | 3 | 4 | 5 | 6 |')
+    print('--------------------------------')
+
+    for row in range(ROWS):
+        rowHeight = ROWS - row
+        currentRow = '|'
+        for col in range(1, COLUMNS + 1):
+            if len(board[col]) < rowHeight:
+                currentRow += '   |'
+            elif board[col][rowHeight - 1] == 'O':
```

Comment:
Well thinking about it a little more, you could potentially remove the duplicate elif statements:

```python
else:
    currentRow += board[col][rowHeight - 1] + " |"
```